requirements.dev.txt

```diff
@@ -2,7 +2,7 @@ bump2version==1.0.1
 flake8==4.0.1
 ipykernel==6.9.0
 jupyter==1.0.0
-mypy==0.941
+mypy==0.981
```

Comment:
This line should not be changed. What justifies this change? Did you encounter any problems when you ran your tests?